README.md

```diff
@@ -204,6 +204,8 @@ git clone git@github.com:zowe/sample-react-app.git
 git clone git@github.com:zowe/sample-iframe-app.git
 ```
 
+**NOTE: react and iframe apps depend on the angular app, as an example of dependencies. Recent versions of the angular app also depends upon ZSS as an example, but you can use a version older than v1.21.0 if you do not have ZSS available.**
+
```

Comment:
Sample React and Iframe Apps depend on the Sample Angular App, as an example of dependencies. Recent versions of the Angular app also depend upon ZSS as an example, but you can use a version older than v1.21.0 if you do not have ZSS available.

Reply:
(capitalization to follow other parts of README)